decidim-core/app/helpers/decidim/menu_helper.rb

```diff
@@ -39,15 +39,15 @@ def breadcrumb_root_menu
       @breadcrumb_root_menu ||= ::Decidim::BreadcrumbRootMenuPresenter.new(
         :menu,
         self,
-        container_options: { class: "menu-bar__main-dropdown__menu" }
```

Comment:
if remove the __main, I would expect the __main class be removed as well here: 
https://github.com/octree-gva/decidim/blob/25d4c388ff09eea4a2767b7677f3c836f48dc061/decidim-core/app/packs/stylesheets/decidim/_header.scss#L394

Reply:
The `__main-dropdown` is used here
https://github.com/octree-gva/decidim/blob/25d4c388ff09eea4a2767b7677f3c836f48dc061/decidim-core/app/views/layouts/decidim/header/_menu_breadcrumb_main_dropdown_desktop.html.erb#L1